crates/bins/src/bin/datadog-static-analyzer-git-hook.rs

```diff
@@ -220,9 +219,16 @@ fn main() -> Result<()> {
         exit(1)
     }
 
-    let configuration_file: Option<ConfigFile> =
-        match get_config(directory_to_analyze.as_str(), use_debug) {
-            Ok(cfg) => cfg,
+    let configuration_file_and_method = get_config(directory_to_analyze.as_str(), use_debug);
+
+    let (configuration_file, configuration_method): (Option<ConfigFile>, Option<ConfigMethod>) =
+        match &configuration_file_and_method {
+            Ok(cfg) => match cfg {
+                Some((config_file, config_method)) => {
+                    (Some(config_file.clone()), Some(config_method.clone()))
+                }
+                _ => (None, None),
```

Comment:
The clones are unnecessary. match on `configuration_file_and_method` (without the reference) since you're not using it after here, and just consume the result without cloning